sdk/storage/Azure.Storage.Queues/tests/QueueTestBase.cs

```diff
@@ -12,6 +12,7 @@
 using Azure.Storage.Sas;
 using Azure.Storage.Test;
 using Azure.Storage.Test.Shared;
+using Microsoft.Extensions.Options;
```

Comment:
NIT - looks like visual studio added this by accident

Reply:
Fixed.